site/_layouts/doc.liquid

```diff
@@ -136,6 +136,7 @@
                     </li>
                     <li {% if page.url == "/docs/v1/tech/apis/two-factor/" %}class="active"{% endif %}><a href="/docs/v1/tech/apis/two-factor/">Multi-Factor/Two Factor</a></li>
                     <li {% if page.url == "/docs/v1/tech/apis/passwordless/" %}class="active"{% endif %}><a href="/docs/v1/tech/apis/passwordless/">Passwordless</a></li>
+                    <li {% if page.url == "/docs/v1/tech/apis/registrations/" %}class="active"{% endif %}><a href="/docs/v1/tech/apis/prometheus/">Prometheus</a></li>
```

Comment:
the `page.url ==` check should be for `/docs/v1/tech/apis/prometheus/`, if you click test in the docs, you'll see that the page does not get highlighted in the nav until this is correct.

Reply:
I think this should go under the System APIs along with the Status endpoint? Any other thoughts?

Reply:
yeah, I agree. I will move it there instead of a separate API doc.